### ISSUE TITLE
Wrong colors in neovim terminal

### ISSUE THREAD

**Issue by ngmoviedo** [sainnhe/gruvbox-material#62]
The colors of neovim's built-in terminal do no match the expected prompt colors. May be a recurrence of #24. It's only happening in neovim (not in vim).

- Operating system/version: Arch Linux
- Terminal emulator/version: kitty 0.19.2
- Shell: fish, version 3.1.2
- `echo $TERM`: xterm-kitty
`nvim --version`:

```
NVIM v0.4.4
Build type: Release
LuaJIT 2.0.5
Compilation: /usr/bin/cc -D_FORTIFY_SOURCE=2 -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 -O2 -DNDEBUG -DMIN_LOG_LEVEL=3 -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wmissing-prototypes-Wimplicit-fallthrough -Wvla -fstack-protector-strong -fno-common -fdiagnostics-color=always -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -I/build/neovim/src/build/config -I/build/neovim/src/neovim-0.4.4/src -I/usr/include -I/build/neovim/src/build/src/nvim/auto -I/build/neovim/src/build/include
Compiled by builduser

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/share/nvim"

Run :checkhealth for more info

```

### Minimal init.vim that can reproduce this bug

```
call plug#begin(stdpath('data') . '/plugged')
Plug 'sainnhe/gruvbox-material'
call plug#end()

set termguicolors
set background=dark
let g:gruvbox_material_background = 'medium'
colorscheme gruvbox-material
```

### Steps to reproduce this bug using minimal vimrc

Launch neovim from the terminal and execute the `:terminal` command.

### Actual behaviour

The prompt has messed up colors:
![wrong](https://user-images.githubusercontent.com/61010998/102238215-cb151c00-3ef5-11eb-8adf-edf64d90b9a9.png)

### Expected behaviour

The prompt should have these colours:
![right](https://user-images.githubusercontent.com/61010998/102238300-e1bb7300-3ef5-11eb-9fbb-73ef469b0371.png)

**Comment by ngmoviedo** [comment]
Further investigation is leading me to think that the error may be on neovim's side. I don't think it's kitty as I have recently tried it also in xterm, getting the same result. It looks similar to [this neovim issue](https://github.com/neovim/neovim/issues/13036#issue-714076513) or [this one](https://github.com/neovim/neovim/issues/4696#issue-152630595)

**Comment by subnut** [comment]
@ngmoviedo It **is** a neovim issue. It has already been fixed by neovim/neovim#10994

Please close this issue
